web/src/compose_notifications.ts

```diff
@@ -109,9 +113,9 @@ export function get_muted_narrow(message) {
     return undefined;
 }
 
-export function get_local_notify_mix_reason(message) {
-    const $row = message_lists.current.get_row(message.id);
-    if ($row.length > 0) {
+export function get_local_notify_mix_reason(message: Message): string | undefined {
+    const $row = message_lists.current?.get_row(message.id);
+    if ($row === undefined || $row.length > 0) {
```

Comment:
This logic is confusing in that it treats `undefined` differently from empty. Probably the best fix is just `assert(message_lists.current !== undefined)`.

Reply:
Yeah definitely doing a pass to assert the message lists as not undefined, in all the PRs that built off of that. Hadn't gotten to this one yet bc was waiting for other stuff to merge. Thanks!